examples/isotopes.ipynb

```diff
@@ -12,7 +12,12 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {},
+   "metadata": {
```

Comment:
Can you clean up the notebook? I think you need to run something like:

```bash
jupyter nbconvert --ClearMetadataPreprocessor.enabled=True isotopes.ypynb
```

Reply:
```
jupyter nbconvert --ClearMetadataPreprocessor.enabled=True --to notebook isotopes.ipynb
```